internal/sourceanalysis/__snapshots__/integration_test.snap

```diff
@@ -227,7 +227,7 @@
             "filename": "\u003cAny value\u003e",
             "offset": -1,
             "line": -1,
-            "column": 16
+            "column": 18
```

Comment:
I believe this is flipping back and forward because in `go.mod` (and thus locally) we're using Go v1.23, but CI is set to use `stable` which is now Go v1.24.

It looks like the Go versioning policy has been removed from the readme at some point, so not sure if we're intending to update to Go v1.24 now, or if we want to switch CI to use `oldstable`...

Reply:
Can we just mock this value so we don't store the column. Let's stick with using stable here.

Reply:
Yeah probably - I think there's also a go vuln reported/not reported by this but we should be able to just ignore that too